addons/website_crm_partner_assign/controllers/main.py

```diff
@@ -133,16 +133,23 @@ def portal_my_opportunities(self, page=1, date_begin=None, date_end=None, sortby
         if date_begin and date_end:
             domain += [('create_date', '>', date_begin), ('create_date', '<=', date_end)]
         # pager
-        opp_count = CrmLead.search_count(domain)
+        opp_count = CrmLead.sudo().search_count(domain)
```

Comment:
I guess this would still be off with "sudo" actually as it won't apply filtering :/

Would need to use `len(opportunities_query)` instead

Reply:
In case of portal, if we search for activities, we could imagine doing a separate search as sudo on ('activity_ids.user_id', '=', portal_user.id), filter out those we want to keep depending on filter, then adding [('id', 'in', found.ids)] in the search_count. Normally you don't have 2 thousands waiting activities as portal user (and anyway we can set a limit).